Sources/StreamChat/Workers/MessageUpdater.swift

```diff
@@ -277,8 +311,35 @@ class MessageUpdater: Worker {
         messageId: MessageId,
         completion: ((Error?) -> Void)? = nil
     ) {
-        apiClient.request(endpoint: .deleteReaction(type, messageId: messageId)) {
-            completion?($0.error)
+        var reaction: MessageReactionDTO?
+
+        database.write { session in
```

Comment:
`NSManagedObjects` are not thread-safe. If we need access to the reaction DTO after the API call we need to open another db session calling `database.write` 👍